src/Form/FormAnalyzer.js

```diff
@@ -175,7 +175,7 @@ class FormAnalyzer {
     }
 
     evaluateUrl () {
-        const path = window.location.pathname
+        const path = window.location.href
```

Comment:
Just simple to check the whole `href`. Imagine if query parameters have login info `?login` or something. I don't think this would break anything, but lemme know.

Reply:
This will probably capture too much - 'href' includes the entire thing (including scheme, origin, ports etc). If you really want to test the combination pathname + search, you might have to create the string manually with  `location.pathname + location.search`.

Also since size of the `hash` part of a URL is basically unlimited (since it's not sent to a server), it might be worth introducing a limit of how much of it you include. `location.pathname + location.search.slice(0, 100)` or something along those lines.

Reply:
I just wanted to have a simpler solution, but good point about slicing the search. If we do limit it to current chase.com problem, we should get away with `window.location.pathname` + `window.location.hash` (which was my original idea.